Dockerfile

```diff
@@ -38,7 +40,3 @@ RUN apt-get update -y \
   && ./aws/install \
   && rm -r ./aws awscliv2.zip \
   && curl -sL https://aka.ms/InstallAzureCLIDeb | bash
-
-FROM ${FINAL_CONTAINER} 
-LABEL maintainer="maintainer@cilium.io"
-WORKDIR /root/app
```

Comment:
Moving this into the 2 target images cilium-cli and cilium-cli-ci makes it possible to cache these layers more efficiently meaning those layers will never have to be redownloaded as it is only the layers with binaries at the end that change.

Reply:
#2817 IMHO that was a poor mans solution to select a specific target in the Dockerfile, maybe the author wasn't aware of the --target build option.

Reply:
ok let's do it